app/src/main/java/com/android/streetworkapp/ui/navigation/NavigationActions.kt

```diff
@@ -0,0 +1,85 @@
+package com.android.streetworkapp.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.AccountCircle
+import androidx.compose.material.icons.outlined.Place
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavHostController
+
+object Route {
+  const val AUTH = "Auth"
+  const val MAP = "Map"
+  const val PROFILE = "Profile"
+  const val UNK = "TBD" // TODO: not yet defined
+}
+
+object Screen {
+  const val AUTH = "Auth Screen"
+  const val MAP = "Map Screen"
+  const val PROFILE = "Profile Screen"
+  const val UNK = "TBD Screen" // TODO: not yet defined
+}
+
+data class TopLevelDestination(val route: String, val icon: ImageVector, val textId: String)
```

Comment:
It could be a good idea to comment this data class too to know what the parameters are for.